x-pack/plugins/search_playground/server/utils/handle_stream_response.ts

```diff
@@ -54,15 +54,15 @@ export const handleStreamResponse = async ({
 
       push(decodedValue);
 
-      await timeout(Math.floor(Math.random() * maxTimeoutMs));
```

Comment:
@yansavitski do you know the purpose of this line? Looks like its delaying until the next chunk but I cant understand why we want to do this?

Reply:
Just to create a small lack in generating of response. Got this line from example of observability team

Reply:
ok got it. i think we should remove it as delaying every chunk.